### PR TITLE
7882 add info layer

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

drag-infolayer

cc @viddo @saleiva @urbanphes

### CODE REVIEW

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 1]
```diff
@@ -15,6 +15,7 @@
   border: 1px solid $cSecondaryLine;
   border-radius: $halfBaseSize;
   background: #FFF;
+  cursor: -webkit-grab;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: only for webkit?

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 2]
```diff
@@ -57,10 +58,14 @@
   left: 24px !important; // compensate for the draggable handle, !important to override inline style="left: 0 ...
   min-width: 290px;
 }
+.Editor-ListLayer .ui-sortable-helper,
+.Editor-newLayerContainer .Editor-ListLayer-item {
+  cursor: -webkit-grabbing;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: Same here.

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 3]
```diff
@@ -57,10 +58,14 @@
...
+}
 .Editor-ListLayer-item.ui-draggable-dragging:hover {
   border: 1px solid rgba($cBlue, 0.48);
   box-shadow: 0 8px 16px rgba(#000, 0.16);
-  cursor: move;
+  cursor: -webkit-grabbing;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: And here! I'm with you HOUND!

File: lib/assets/javascripts/cartodb3/editor/layers/layer-analysis-draggable-view.js [hunk 4]
```diff
@@ -10,7 +11,7 @@ var analyses = require('../../data/analyses');
  */
 module.exports = CoreView.extend({
   options: {
-    bgColor: '#E27D61',
+    getNextLetter: function () { return 'x'; },
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-analysis-draggable-view.js]: Why do we change this thing? I guess @viddo should review this.

Comment (viddo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-analysis-draggable-view.js]: I did that change… 😉 
With this change the dragged element appears with the same ("next") letter and color representation when the layer is actually created.

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-analysis-draggable-view.js]: :joy:, ok, no much to say.

**Comment by xavijam** [comment]
Merge when @piensaenpixel makes a small change and we have 🍏  light.

**Comment by piensaenpixel** [comment]
@xavijam sorry for hound but... we need add some vendors :P, take a look and if all is ok we can merge it.

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 5]
```diff
@@ -190,7 +201,9 @@
   background: $cThirdBackground;
   &:hover {
     border: 1px solid $cSecondaryLine;
-    cursor: default;
+  cursor: move;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: Hey!

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 6]
```diff
@@ -190,7 +201,9 @@
...
+  cursor: -webkit-grab;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: Line indentation, HEY!

**Pull request merged**